keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -123,14 +123,14 @@
           "Keys"
         ],
         "operationId": "DeleteKey",
-        "description": "Deletes the specified key.",
+        "description": "Deletes a key of any type from storage in Azure Key Vault. The delete key operation cannot be used to remove individual versions of a key. This operation removes the cryptographic material associated with the key, which means the key is not usable for Sign/Verify, Wrap/Unwrap or Encrypt/Decrypt operations. Authorization: Requires the keys/delete permission",
```

Comment:
The doc team made a rule that all the descriptions should start with an upper case letter and must end with a full stop. Please make sure that the rule is honored in all the descriptions.